includes/Documents/Invoice.php

```diff
@@ -557,7 +557,7 @@ public function get_pdf_settings_fields( $option_name ) {
 			$settings_fields[] = array(
 				'type'     => 'setting',
 				'id'       => 'include_link_guest_emails',
```

Comment:
Perhaps we should consider omitting "guest" from the slugs and function names as well.